xml/hwbp_registers.xml

```diff
@@ -270,23 +285,86 @@
             by 1.
         </field>
 
-        <field name="action" bits="5:0" access="R/W" reset="0">
-            Determines what happens when this trigger matches.
-
-            0: Raise a breakpoint exception. (Used when software wants to use the
-            trigger module without an external debugger attached.)
+        <field name="action" bits="5:0" access="R/W" reset="0" />
+    </register>
 
-            1: Enter Debug Mode. (Only supported when \Fdmode is 1.)
+    <register name="Exception Trigger" short="etrigger" address="0x7a1">
+        This register is accessible as \Rtdataone when \Ftype is 4.
 
-            2: Start tracing.
+        If implemented, it allows triggering on a variety of interrupt and
+        exception sources. If such a trigger is taken, all the usual CSRs as
```

Comment:
I think "the usual CSRs" is a little vague. Why not say CSRs are updated as usual as described by the RISC-V Priv Arch Manual Section XYZ (we should be pointing to a specific version of the spec anyway).

Reply:
I've referred to the doc. I don't think the version is relevant until they standardize something.